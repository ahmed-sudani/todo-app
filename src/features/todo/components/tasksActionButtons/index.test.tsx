import { describe, expect, it } from 'vitest'
import TasksActionButtons from '.'

import { render } from '@testing-library/preact'

describe('TasksActionButtons', () => {
  it('should render successfully', () => {
    const { container } = render(<TasksActionButtons />)
    expect(container).toMatchSnapshot()
  })
})
