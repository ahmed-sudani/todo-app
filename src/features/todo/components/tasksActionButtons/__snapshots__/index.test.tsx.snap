// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TasksActionButtons > should render successfully 1`] = `
<div>
  <div
    class="flex gap-1 sm:justify-end justify-start items-center"
  >
    <button
      class="btn btn-rounded btn-primary"
    >
      Complete All Tasks
    </button>
    <button
      class="btn btn-rounded btn-danger"
    >
      Delete Completed Tasks
    </button>
  </div>
</div>
`;
